api/test/docker/apisix_config2.yaml

```diff
@@ -22,7 +22,7 @@ etcd:
     - "http://172.16.238.10:2379"
     - "http://172.16.238.11:2379"
     - "http://172.16.238.12:2379"
-  resync_delay: 0.1  # sync data from etcd quickly for e2e test
+  resync_delay: 0  # sync data from etcd quickly for e2e test
```

Comment:
0 is ok？

Reply:
sure, you could see:
https://github.com/apache/apisix/blob/6a986f46490fa16166b83513ecde85555b6fe1b2/apisix/cli/ops.lua#L257
https://github.com/apache/apisix/blob/92b352862a7770daa0b1ac50893c4c68d84af877/apisix/core/config_etcd.lua#L593-L596
and
https://github.com/apache/apisix/blob/92b352862a7770daa0b1ac50893c4c68d84af877/apisix/core/config_etcd.lua#L563-L574

if we set 1 , will cause unstable.

Reply:
ping @starsz

Reply:
Got it.Thank you for your explanation.